torch/_dynamo/variables/dicts.py

```diff
@@ -716,13 +704,6 @@ def call_method(
 
         unimplemented("custom dict: call_method unimplemented name=%s", name)
 
-    def var_getattr(self, tx, name: str) -> "VariableTracker":
```

Comment:
This override was only needed for dataclass-like custom dicts (and it was actually a source of unsoundness, because attempting to access a normal attribute of a non-dataclass custom dict class would have incorrectly attempted to rewrite it into a `__getitem__` call)